schemas/2015-08-01/Microsoft.Web.json

```diff
@@ -0,0 +1,389 @@
+                                },
+                                {
+                                    "enum": [
+                                        "Free",
+                                        "Shared",
+                                        "Basic",
+                                        "Standard"
+                                    ]
+                                }
+                            ],
+                            "description": "Microsoft.Web/serverfarms: Server farm sku."
+                        },
+                        "workerSize": {
+                            "oneOf": [
+                                {
+                                    "$ref": "http://schema.management.azure.com/schemas/2015-05-01/deploymentTemplate.json#/definitions/expression"
+                                },
+                                {
+                                    "enum": [
+                                        "Small",
+                                        "Medium",
+                                        "Large"
+                                    ]
+                                },
+                                {
+                                    "type": "integer",
+                                    "minimum": 0,
+                                    "maximum": 2
+                                }
+                            ],
+                            "description": "Microsoft.Web/serverfarms: The instance size."
+                        },
+                        "numberOfWorkers": {
+                            "oneOf": [
+                                {
+                                    "$ref": "http://schema.management.azure.com/schemas/2015-05-01/deploymentTemplate.json#/definitions/expression"
+                                },
+                                {
+                                    "type": "integer",
+                                    "minimum": 0,
+                                    "maximum": 10
+                                }
+                            ],
+                            "description": "Microsoft.Web/serverfarms: The instance count, which is the number of virtual machines dedicated to the farm. Supported values are 1-10."
+                        }
+                    }
+                }
+            },
+            "required": [
+                "type",
+                "apiVersion",
+                "properties"
+            ]
+        },
+        "config": {
+            "type": "object",
+            "properties": {
+                "type": {
+                    "enum": [
+                        "Microsoft.Web/sites/config",
+                        "config"
+                    ]
+                },
+                "apiVersion": {
+                    "enum": [
+                        "2015-08-01"
+                    ]
+                },
+                "name": {
```

Comment:
Do you really need this? all three configs "web, connectionstring, appsettings" has name property already and they have different schema with this one.